### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by arist76**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Mindplex Token
+## Overview
+The MindplexToken contract is an ERC20 utility token to be used in the Mindplex platform. Additional functionalities include pausing and unpausing transfers, burning and minting tokens, and role-based access control for certain operations. This contract leverages OpenZeppelin libraries to implement these features securely and efficiently.
+
+## Features
+- **Fixed Supply**: The token has a fixed maximum supply to be determined on deployment.
+- **Pausable**: The contract allows pausing and unpausing of all token transfers.
+- **Burnable**: Tokens can be burned, reducing the total supply.
+- **Mintable**: New tokens can be minted up to the maximum supply.
```
Comment (Timekiller7) [inline README.md:9]: in the current implementation we mint all amount of tokens during deployment process

File: README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- **Access Control**: Specific roles are defined for pausing, unpausing, and minting tokens.
+- **Votes**: Integration with ERC20Votes for governance functionalities.
+## Roles
+- **Admin Role (DEFAULT_ADMIN_ROLE)**: Has the highest level of control over the contract, including managing other roles.
+**Pauser Role (PAUSER_ROLE)**: Can pause and unpause token transfers.
+**Minter Role (MINTER_ROLE)**: Can mint new tokens up to the maximum supply.
```
Comment (Timekiller7) [inline README.md:15]: we can remove it, since we mint all amount of tokens during deployment process

**Comment by Timekiller7** [comment]
Waiting for PR to be fixed according to new requirements due to changes in tokenomics

**Comment by arist76** [comment]
I have made the required changes based on the changes made in PR #5.

**Review by Timekiller7 — APPROVED** [review]

**Review by Timekiller7 — APPROVED** [review]

**Comment by Timekiller7** [comment]
@arist76, could you add implementation description? Like you did in Mindplex Escrow

**Comment by arist76** [comment]
Ok will do 👍

**Comment by arist76** [comment]
> 

I have made the changes. check it out and let me know what needs to change.

**Pull request merged**